### ISSUE TITLE
Aimbot on KMbox NET Not working properly

### ISSUE THREAD

**Issue by RyanAMTB** [Enoouo/Pro-CS2_DMA#71]
Hello Enoouo, 

I have been using this cheat for a while now and I have had no aimbot. When the aimbot does work, the camera pans and skews harshly in different directions, not even close to an enemy. Do you think it could be an issue with my KMBox NET? My windows version is here.

WIndows 10 22H2 19045.5011

Thanks for reading, and great release regardless

**Comment by 13Gizmo12** [comment]
have you made it work?

**Comment by RyanAMTB** [comment]
> have you made it work?
> 
> 

Yes update Client.json and Offsets.json and then apply your KMBOX net settings into the GUI. The issue was Client.json. Get that information by looking up CS2Dumper on GitHub and then look for the latest dump.
